packages/app-project/src/screens/SubjectTalkPage/components/SubjectTalkData/components/Mentions/Mentions.jsx

```diff
@@ -0,0 +1,99 @@
+import { Box } from 'grommet'
+import { BlockQuote } from 'grommet-icons'
+import { useTranslation } from 'next-i18next'
+import { arrayOf, shape, string } from 'prop-types'
+import styled from 'styled-components'
+
+import ParticipantsAndComments from '../ParticipantsAndComments'
+import SectionHeading from '../SectionHeading'
+import TalkComment from '../TalkComment'
+
+const StyledMentions = styled(Box)`
+  max-height: auto;
+
+  @media screen and (max-width: 1280px) {
+    max-height: 400px;
+  }
+`
+
+const StyledOrderedList = styled(Box)`
+  list-style: none;
+`
+
+function Mentions({
+  mentions = [],
+  roles = [],
+  users = []
+}) {
+  const { t } = useTranslation('screens')
+
+  return (
+    <StyledMentions
+      border={{ color: 'light-5', side: 'top', size: '0.5px' }}
+      gap='xsmall'
+      pad='small'
+    >
+      <Box
+        align='center'
+        direction='row'
+        gap='small'
+      >
+        <SectionHeading
+          icon={
+            <BlockQuote
+              color={{ dark: 'light-1', light: 'dark-4' }}
+              size='1rem'
+            />
+          }
+          title={t('Talk.mentions')}
+        />
+        {mentions?.length > 0 && (
+          <ParticipantsAndComments
+            commentsCount={mentions.length}
+            usersCount={mentions.length}
+          />
```

Comment:
The comments count and users count will always be the same, as a single user creates a related mention comment. I think that's ok, but noting just in case we want to reconsider.